eZ/Publish/API/Repository/Tests/SearchEngineIndexingTest.php

```diff
@@ -338,6 +338,106 @@ public function testDeleteLocation()
     }
 
     /**
+     * Test content is available for search after being published.
+     */
+    public function testPublishVersion()
+    {
+        $repository = $this->getRepository();
+
+        $contentService = $repository->getContentService();
+        $locationService = $repository->getLocationService();
+        $searchService = $repository->getSearchService();
+
+        $contentType = $this->createDeletionTestContentType();
+        $contentTypeStruct = $contentService->newContentCreateStruct($contentType, 'eng-GB');
+        // create random value to avoid conflicts with other tests
+        $contentFieldValue = uniqid();
+        $contentTypeStruct->setField('name', $contentFieldValue);
+
+        $parentLocationList = [$locationService->newLocationCreateStruct(2)];
+
+        $contentDraft = $contentService->createContent($contentTypeStruct, $parentLocationList);
```

Comment:
At first I wanted here an independent test, but giving it more thought, yes, it's better to reuse `createContentWithName`. Fixed in 477211c.